packages/core/theme-module/theme/pages/Product.vue

```diff
@@ -168,36 +168,11 @@
         </SfSticky>
       </div>
     </div>
-    <SfSection title-heading="Match it with" class="section">
-      <SfCarousel class="product-carousel">
-        <SfCarouselItem v-for="(product, i) in products" :key="i">
-          <SfProductCard
-            :title="product.title"
-            :image="product.image"
-            :regular-price="product.price.regular"
-            :max-rating="product.rating.max"
-            :score-rating="product.rating.score"
-            :is-on-wishlist="product.isOnWishlist"
-            class="product-card"
-          />
-        </SfCarouselItem>
-      </SfCarousel>
-    </SfSection>
-    <SfSection title-heading="You might also like" class="section">
-      <SfCarousel class="product-carousel">
-        <SfCarouselItem v-for="(product, i) in products" :key="i">
-          <SfProductCard
-            :title="product.title"
-            :image="product.image"
-            :regular-price="product.price.regular"
-            :max-rating="product.rating.max"
-            :score-rating="product.rating.score"
-            :is-on-wishlist="product.isOnWishlist"
-            class="product-card"
-          />
-        </SfCarouselItem>
-      </SfCarousel>
-    </SfSection>
+    <RelatedProducts
+      v-if="product"
+      :product="product"
+      title="Match it with"
+    />
```

Comment:
short explanations for the rest of the team: I suggested to display just one section of related products instead of two because we had no idea what to display, what searching criteria we can use - the first section displays just products from the same category as a product you visited